### PR TITLE
Trev8939/line of sight geo element

### PR DESCRIPTION
**Pull request by TADraeseke**
Thanks for the other review. Can I trouble you for one more @priy1289?

Thanks!

### CODE REVIEW

File: java/line-of-sight-geoelement/src/main/java/com/esri/arcgisruntime/sample/lineofsightgeoelement/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+    // create a point graph near the Empire State Building to be the observer
+    Point observationPoint = new Point(-73.9853, 40.7484, 200, SpatialReferences.getWgs84());
+    Graphic observer = new Graphic(observationPoint,
+        new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.CIRCLE, 0xFFFF0000, 5));
+    graphicsOverlay.getGraphics().add(observer);
+
+    // create a slider to change the observer's Z value
+    SeekBar heightSeekBar = findViewById(R.id.heightSeekBar);
+    TextView currHeightTextView = findViewById(R.id.currHeightTextView);
+    heightSeekBar.setMax(150);
```
Comment (priy1289) [inline java/line-of-sight-geoelement/src/main/java/com/esri/arcgisruntime/sample/lineofsightgeoelement/MainActivity.java]: extract to an integer variable?

File: java/line-of-sight-geoelement/src/main/java/com/esri/arcgisruntime/sample/lineofsightgeoelement/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+    heightSeekBar.setOnSeekBarChangeListener(new SeekBar.OnSeekBarChangeListener() {
+      @Override public void onProgressChanged(SeekBar seekBar, int progress, boolean b) {
+        // offset progress bar to go from 150 - 300
+        int height = progress + 150;
```
Comment (priy1289) [inline java/line-of-sight-geoelement/src/main/java/com/esri/arcgisruntime/sample/lineofsightgeoelement/MainActivity.java]: here too..

File: java/line-of-sight-geoelement/src/main/java/com/esri/arcgisruntime/sample/lineofsightgeoelement/MainActivity.java [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+      @Override public void onStartTrackingTouch(SeekBar seekBar) {
+
+      }
+
+      @Override public void onStopTrackingTouch(SeekBar seekBar) {
+
+      }
+    });
+    // set seek bar initial progress with offset
+    heightSeekBar.setProgress((int) observationPoint.getZ() - 150);
```
Comment (priy1289) [inline java/line-of-sight-geoelement/src/main/java/com/esri/arcgisruntime/sample/lineofsightgeoelement/MainActivity.java]: same here..

**Comment by TADraeseke** [comment]
Good idea to make that int variable. Thanks for the review @priy1289!

**Pull request merged**